### PR TITLE
Feature/script

### PR DESCRIPTION
**Pull request by reardonj**

### CODE REVIEW

File: characters.txt [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+Andrew - Tax collector 
+Initial state
+- Outraged that you would delay him 
+
+1st talk option
+- Andrew bellows at you that he is a collector for the NTA (Nottingham Tax Association) and that you are costing the region valuable dollars by delaying him. If you continue, he demands recompense equal to his hourly wages.
+
+2nd talk option
+- Andrew demands to speak to your superior. While yelling, your superior comes over and confirms that Andrew is in fact a tax collector. He tells you not to waste anymore time and to let Andrew pass.
```
Comment (reardonj) [inline characters.txt:20]: I'd prefer to leave this as a "If Andrew is still present when time ends, you get reamed out by the sheriff"

File: characters.txt [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+Evan - Stonemason
+Jon - Winemaker
+Mike - Blacksmith
+
+Andrew - Tax collector 
+Initial state
+- Outraged that you would delay him 
+
+1st talk option
+- Andrew bellows at you that he is a collector for the NTA (Nottingham Tax Association) and that you are costing the region valuable dollars by delaying him. If you continue, he demands recompense equal to his hourly wages.
```
Comment (reardonj) [inline characters.txt:17]: should this be the second option? I'd expect him to be civil when he first addresses the player.

File: characters.txt (hunk 1 above)
Comment (mbrie041) [reply characters.txt:20]: Noted. I'll work that in.

File: characters.txt (hunk 2 above)
Comment (mbrie041) [reply characters.txt:17]: Yeah I hadn't considered his initial greeting. I'll make sure to consider that for all the characters moving forward.

**Pull request merged**